waspc/src/Wasp/Psl/Ast/Model.hs

```diff
@@ -60,5 +60,5 @@ data FieldTypeModifier
   | Optional
   deriving (Show, Eq, Data)
 
-getFields :: Model -> [Field]
```

Comment:
This was unused?

Reply:
Yes. I couldn't compile the project if not :)